c++/src/kj/compat/brotli-test.c++

```diff
@@ -404,6 +409,48 @@ KJ_TEST("async brotli huge round trip") {
   KJ_ASSERT(memcmp(bytes.begin(), decompressed.begin(), bytes.size()) == 0);
 }
 
+KJ_TEST("brotli with write flush option") {
+  auto io = setupAsyncIo();
+
+  MockAsyncOutputStream rawOutput;
+  BrotliAsyncOutputStream brotli(rawOutput, BrotliAsyncOutputStream::Options{
+    .writeFlush = true,
+  });
+
+  auto bytes = heapArray<byte>(100);
+  for (auto& b: bytes) {
+    b = rand();
```

Comment:
I worry that this test is non-deterministic – we don't set a seed for rand(), so the input data is arbitrary and we'd expect to sometimes have data that has higher redundancy (e.g. being able to compress by 1 or 2 bytes) and compresses the header and first block to less than 103 bytes. Additionally, a future version of brotli may be able to compress data better (or perform worse in edge cases), so the expected output size could be different after an update too.
Maybe we can just check that there are 12 writes with a non-zero size?